doc/pcieinfo_design.md

```diff
@@ -0,0 +1,81 @@
+## Motivation
+Add a PCIe Diag tool for SONiC. This tool including three commands    
+### Commands
+
+    show platform pcieinfo     -----> Show current device PCIe info
+    show platform pcieinfo -c  -----> Check whether the PCIe info is correct   
+    pcieutil pcie_generate     -----> Generate an PCIe info congfig file
```

Comment:
Should we add a sub command to 'show platform pcieinfo" to generate config file? What is the concern not having it in the main command?